rollup.config.js

```diff
@@ -9,11 +12,18 @@ const camelCase = string => {
 
 const input = 'src/index.js';
 const name = camelCase(pkg.name);
+const external = id => !id.startsWith('.') && !path.isAbsolute(id);
+const commonjsOptions = {
+  namedExports: {
+    'react-is': ['ForwardRef', 'isMemo']
```

Comment:
would do `Object.keys(require('react-is'))` here to be future-proof

Reply:
Ah, sure. I use the same in my prod build :)